tests/test_hermes_provider_parity.py

```
@@ -640,6 +640,127 @@ def test_sync_adapter_pull_tolerates_null_next_cursor(sync_modules):
+    modules = {
+        "hermes_memory_provider": _import_module("hermes_memory_provider", PROJECT_ROOT),
+        "mnemosyne_hermes": _import_module("mnemosyne_hermes", INTEGRATION_SRC),
+    }
+
+    assert {name: module.MnemosyneMemoryProvider.PERSONA_TOKEN_CAP for name, module in modules.items()} == {
+        "hermes_memory_provider": 1500,
+        "mnemosyne_hermes": 1500,
+    }
+
+
+def test_packaged_provider_import_survives_missing_core_helpers(monkeypatch):
```

Ruff (0.15.20)

[warning] Unused function argument: `monkeypatch`

(ARG001)